src/test/scala-2.12/org/apache/james/gatling/simulation/imap/ImapListMessagesBodyStructureSimulation.scala

```diff
@@ -0,0 +1,24 @@
+package org.apache.james.gatling.simulation.imap
+
+import com.linagora.gatling.imap.PreDef.imap
+import io.gatling.core.Predef._
+import io.gatling.core.scenario.Simulation
+import org.apache.james.gatling.control.{UserCreator, UserFeeder}
+import org.apache.james.gatling.imap.scenari.ImapListMessagesBodyStructureScenario
+import org.apache.james.gatling.simulation.{Configuration, HttpSettings}
+
+import scala.concurrent.duration._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, Future}
+
+class ImapListMessagesBodyStructureSimulation extends Simulation {
+
+  private val users = new UserCreator(Configuration.BaseJamesWebAdministrationUrl).createUsersWithInboxAndOutbox(Configuration.UserCount)
```

Comment:
You don't want to create users, you just want to use the existing one already created in the provisioned data.
You can use the users in SimulationOnMailCorpus like it's done on the JMAP simulations